pkg/kv/dynamodb/store.go

```diff
@@ -206,7 +206,7 @@ func (s *Store) Get(ctx context.Context, partitionKey, key []byte) (*kv.ValueWit
 	dynamoRequestDuration.WithLabelValues(operation).Observe(time.Since(start).Seconds())
 	if err != nil {
 		dynamoFailures.WithLabelValues(operation).Inc()
-		return nil, fmt.Errorf("get item: %s (key=%v): %w", err, string(key), kv.ErrOperationFailed)
+		return nil, fmt.Errorf("get item: %w", err)
```

Comment:
I think this one (and all the following that converts `key` into string, actually logs the key in a pretty readable format. Are you sure we want to discard these?